src/models/axis.js

```diff
@@ -84,7 +85,7 @@ nv.models.axis = function() {
                         .attr('y', 0)
                         .attr('x', w/2);
                     if (showMaxMin) {
-                        var axisMaxMin = wrap.selectAll('g.nv-axisMaxMin')
```

Comment:
meant to move the declaration to the top of the switch statement.  It was doing duplicate declarations.  Fixed now.